BodoSQL/calcite_sql/bodosql-calcite-application/src/main/java/org/apache/calcite/sql/type/BodoReturnTypes.java

```diff
@@ -119,12 +119,9 @@ public class BodoReturnTypes {
                 RelDataTypeFactory factory = opBinding.getTypeFactory();
                 RelDataType stringType = factory.createSqlType(SqlTypeName.VARCHAR);
                 RelDataType mapType = factory.createMapType(stringType, stringType);
-                return mapType;
-                // TODO: Replace the above implementation with the below implementation when
```

Comment:
This was added by me and I guess I never got around to applying the commented out code.